packages/fxa-auth-server/lib/routes/subscriptions/apple.ts

```diff
@@ -126,6 +126,7 @@ export class AppleIapHandler {
         case PurchaseUpdateError.INVALID_ORIGINAL_TRANSACTION_ID:
           throw error.iapInvalidToken(err);
         case PurchaseUpdateError.CONFLICT:
+          throw error.iapPurchaseConflict(err);
```

Comment:
Notably: this is now a 4xx instead of a 5xx error. This means it won't be reported by Sentry, though we will still have [error logs](https://github.com/mozilla/fxa/blob/ee71759ff2c282a876c0e7e0ddb3a4e888aefc0d/packages/fxa-auth-server/lib/payments/iap/apple-app-store/purchase-manager.ts#L91-L94) that we can inspect.